src/kontrol/foundry.py

```diff
@@ -309,9 +309,14 @@ def get_test_id(self, test: str, version: int | None) -> str:
         if len(matching_proof_ids) == 0:
             raise ValueError(f'Found no matching proofs for {test}:{version}.')
         if len(matching_proof_ids) > 1:
-            raise ValueError(
-                f'Found {len(matching_proof_ids)} matching proofs for {test}:{matching_proof_ids}. Use the --version flag to choose one.'
-            )
+            if version == None:
+                raise ValueError(
+                    f'Found {len(matching_proof_ids)} matching proofs for {test}:{version}. Use the --version flag to choose one.'
+                )
+            else:
+                raise ValueError(
+                    f'Found {len(matching_proof_ids)} matching proofs for {test}:{version}. Provide full signatures to the test.'
```

Comment:
Do you think we can make the error message a bit more specific? E.g., provide an example of the full test signature (I copied this one from our `kontrol prove --match-test` help message. I'm just a bit worried that users may not really know what's the format of a full signature (plus, it has to be wrapped in '' symbols).
```suggestion
                    f"Found {len(matching_proof_ids)} matching proofs for {test}:{version}. Provide a full signature of the test, e.g., 'ERC20Test.testTransfer(address,uint256)'."
```

Reply:
Looks great, thank you!